library/rocksdbjni/RocksDbBuilder.kt

```diff
@@ -4,11 +4,13 @@ import org.zeroturnaround.exec.ProcessExecutor
 import org.zeroturnaround.exec.ProcessResult
 import java.nio.file.Path
 import java.nio.file.Paths
+import kotlin.test.assertNotNull
```

Comment:
Are we using this?

Reply:
Good spot, removed in https://github.com/graknlabs/dependencies/pull/213/commits/867af6f633a8987ab61a945f505b19bac24dd58b